playground-remote-inference.md

```diff
@@ -0,0 +1,72 @@
+# Remote inference in Playground
+
+Users can chat with external language model in playground.
+
+## Prerequisites
+
+- AI Toolkit v0.4.0 or newer.
+- OpenAI compatible chat completion endpoint.
+
+## How to add a remote model
+
+1. Click the "+" icon when hovering on "MODELS" tree view or enter `AI Toolkit: Add model for remote inference` in command palette.
+
+    ![](./Images/playground-remote-inference/0-entrypoint-treeview.png)
+
+    ![](./Images/playground-remote-inference/1-entrypoint-command.png)
+
+2. Enter the model name and press Enter.
+
+    ![](./Images/playground-remote-inference/2-model-name.png)
+
+    > Note: The model name is sent to the chat completion endpoint as the ["model" parameter](https://platform.openai.com/docs/api-reference/chat/create#chat-create-model) when running inference.
+
+3. Enter OpenAI compatible chat completion endpoint URL.
+
+    ![](./Images/playground-remote-inference/3-endpoint.png)
+
+4. If your endpoint requires authentication, you can set an authentication header for API key authentication. Otherwise, your can just press Enter to skip this step.
```

Comment:
"you can set an authentication header for API key authentication" -> "you can set an authentication header for API key"

Reply:
Done